### DISCUSSION TITLE
Invo example getting 502 bad gateway on session creation

### DISCUSSION THREAD

**Discussion by nmcwilli** [phalcon/cphalcon#15617] [Q&A] [answered]
I just tested the INVO example and everything is working, except session creation and login. 

I am getting error 502 bad gateway when trying to run startAction() at url /session/start 

all other controllers and pages function fine. 

With the invo sample code, is there any adjustments required in the router or DI set to make this work?

**Comment by quasipickle** [comment]
Are you able to check the server error logs?

**Comment by nmcwilli** [comment, marked as answer]
Sorry, forgot to update this discussion thread. Thanks for your response. I got the problem resolved! It appeared that I had a zend_mm_corruption in my php error log and some of the extensions were not functioning properly. I did a fresh new build from scratch on a new server and was able to get up and running.
